src/include/columnar/columnar_compression.h

```diff
@@ -0,0 +1,36 @@
+/*-------------------------------------------------------------------------
+ *
+ * columnar_compression.h
+ *
+ * Type and function declarations for compression methods.
+ *
+ * Copyright (c) 2016, Citus Data, Inc.
+ *
+ * $Id$
```

Comment:
Looked into other files in codebase, I guess we should not have `$Id$` and copyright years anymore

```suggestion
 * Copyright (c) Citus Data, Inc.
```

Reply:
Done.